src/components/vistas/confirmation/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styles from './styles.module.scss';
+import { Link } from 'react-router-dom';
+
+function Confirmation() {
+  return (
+    <div className={styles.contenedorPrincipal}>
+      <div className={styles.containerText}>
+        <h1 className={styles.title}>¡Todo listo!</h1>
+        <p >La Fundacion Hablemos de HIV te está esperando.</p>
```

Comment:
Atento a los espacios de más. También, ¿no debería tener este elemento la clase `text`?
```suggestion
        <p>La Fundacion Hablemos de HIV te está esperando.</p>
```

Reply:
se me paso, listo !